stable/cockroachdb/values.yaml

```diff
@@ -202,6 +202,22 @@ statefulset:
     #   cpu: 100m
     #   memory: 512Mi
 
+  # Controls settings for the liveness probe. We recommend against using a
```

Comment:
Link GH issue where this was discussed?

Reply:
Added a link to where this was discussed.